Modules/CTP/include/CTP/RawDataReaderCheck.h

```diff
@@ -65,16 +65,16 @@ class RawDataReaderCheck : public o2::quality_control::checker::CheckInterface
+  std::vector<int> mVecGoodBC;                                                                                                                                                                                                                                                                                                                                           // vector of good BC positions
+  std::vector<int> mVecMediumBC;                                                                                                                                                                                                                                                                                                                                         // vector of medium BC positions, we expect a BC at this position, but inputs are below mThreshold
+  std::vector<int> mVecBadBC;                                                                                                                                                                                                                                                                                                                                            // vector of bad BC positions, we don't expect a BC at this position, but inputs are abow mThreshold
+  std::vector<int> mVecIndexBad;                                                                                                                                                                                                                                                                                                                                         // vector of ctp input and class indices, which had a big relative change
+  std::vector<int> mVecIndexMedium;                                                                                                                                                                                                                                                                                                                                      // vector of ctp input and class indices, which had a relative change
+  std::bitset<o2::constants::lhc::LHCMaxBunches> mLHCBCs;                                                                                                                                                                                                                                                                                                                // LHC filling scheme
+  const char* ctpinputs[49] = { "T0A", "T0C", "TVX", "TSC", "TCE", "VBA", "VOR", "VIR", "VNC", "VCH", "11", "12", "UCE", "DMC", "USC", "UVX", "U0C", "U0A", "COS", "LAS", "EMC", "PH0", "23", "24", "ZED", "ZNC", "PHL", "PHH", "PHM", "30", "31", "32", "33", "34", "35", "36", "EJ1", "EJ2", "EG1", "EG2", "DJ1", "DG1", "DJ2", "DG2", "45", "46", "47", "48", "49" }; // ctp input names
```

Comment:
The inputs already exists in:
https://github.com/AliceO2Group/AliceO2/blob/bdde90491b05acd2505b67be723eda13033be949/DataFormats/Detectors/CTP/src/Configuration.cxx#L1071
Can not this beused ?